tests/test_minimal/test_tools/globus_transfer_tools.py

```diff
@@ -15,14 +16,9 @@
     transfer_globus_content,
 )
 
-try:
```

Comment:
This was the only thing that could not be fixed automatically; but it's a good catch

Reply:
This is better for the debugger as well. When I debug I need to skip over those assertions so the change here is better.